common-npm-packages/docker-common-v2/make.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I tried to replicate the generation of the local package - similar to the utility-common-v2 - such that the docker-common-v2 could use my local code as opposed to a publicly deployed npm package - as previously done with the prior version of the code. In this way, I had to introduce this file so that the code that I had made code be zipped correctly for testing purposes inside of ADO.

Reply:
I assume this is for testing and you will revert this before merge ?